test/server.js

```diff
@@ -0,0 +1,10 @@
+
+var request = require("request");
+var assert = require('assert');
+
+it("returns status code 200", function(){
+    request.get("http://localhost:8080/", function(err, response, body){
+        assert.equal(200, response.statusCode);
+        done();
+    });
```

Comment:
This looks more like an integration test, can we instead change it to unit test. I don't want to start the server but test everything inside the file.